src/main/java/com/hotels/hcommon/ssh/SshSettings.java

```diff
@@ -54,7 +56,7 @@ public Builder withRoute(@TunnelRoute String route) {
       return this;
     }
 
-    public Builder withPrivateKeys(@NotNull String privateKeys) {
+    public Builder withPrivateKeys(String privateKeys) {
```

Comment:
Why does this no longer have the `@NotNull` annotation? Is that because some usages of this in other projects can have this null?

Reply:
Ah my bad, that's left from while I was testing for something else; I'll put it back.